databases/relational-databases.md

```diff
@@ -31,6 +32,36 @@ To understand, query, and insert information into a relational database, a techn
 
 ### Specific Things To Teach
 
+- The specific problems a database solves
```

Comment:
Please make sure all of this info is provided later on in the outline. If it isn’t, please add it.